### ISSUE TITLE
False positive in require-timeout-settings rule for CREATE TABLE statements

### ISSUE THREAD

**Issue by ermakov-oleg** [sbdchd/squawk#673]
## Description

The new `require-timeout-settings` rule introduced in v2.28.0 is producing false positives for simple `CREATE TABLE` statements that don't actually require timeout settings.

## Expected Behavior

Simple `CREATE TABLE` statements should not trigger timeout warnings since they are typically fast operations that don't cause significant locking issues.

## Actual Behavior

The rule flags all `CREATE TABLE` statements as requiring both `lock_timeout` and `statement_timeout` settings, even for simple table creation.

## Example

**Migration file:**
```sql
create table foo (
    id integer generated by default as identity primary key
);
```

**Squawk output:**
```
warning[require-timeout-settings]: Missing `set lock_timeout` before potentially slow operations
  ╭▸ ff.sql:1:1
  │
1 │ ┏ create table foo (
2 │ ┃     id integer generated by default as identity primary key
3 │ ┃ );
  │ ┗━┛
  │
  ├ help: Configure a `lock_timeout` before this statement.
  ╭╴
1 + set lock_timeout = '1s';
  ╰╴
warning[require-timeout-settings]: Missing `set statement_timeout` before potentially slow operations
  ╭▸ ff.sql:1:1
  │
1 │ ┏ create table foo (
2 │ ┃     id integer generated by default as identity primary key
3 │ ┃ );
  │ ┗━┛
  │
  ├ help: Configure a `statement_timeout` before this statement
  ╭╴
1 + set statement_timeout = '5s';
  ╰╴

Find detailed examples and solutions for each rule at https://squawkhq.com/docs/rules
Found 2 issues in 1 file (checked 1 source file)
```

## Analysis

Creating a new table is generally a lightweight operation that:
- Doesn't modify existing data
- Doesn't acquire locks on existing tables
- Completes quickly in most cases

The timeout settings should probably be required for operations that are actually slow or potentially blocking, such as:
- `ALTER TABLE` with data modifications
- Large `INSERT`/`UPDATE`/`DELETE` operations
- Adding constraints to existing tables with data

**Comment by sbdchd** [comment]
Should be fixed in latest, thank you for the report!
